src/main/java/seedu/exercise/ui/LineChartPanel.java

```diff
@@ -58,5 +59,11 @@ private void display() {
         lineChart.setLegendVisible(false);
         lineChart.setTitle(ChartTextUtil.lineAndBarChartTitleFormatter(category, startDate, endDate));
         lineChart.getData().add(series);
+
+        series.getData().stream().forEach(data -> {
+            Tooltip tooltip = new Tooltip();
+            tooltip.setText(data.getXValue() + "\n" + data.getYValue());
+            Tooltip.install(data.getNode(), tooltip);
+        });
```

Comment:
Since this methods are repeated in multiple chart panels, maybe have them all abstracted out into one method in a util class or a superclass

Reply:
ok